manifest.php

```diff
@@ -37,12 +37,10 @@
     'install' => array(
         'rdf' => array(
             __DIR__. '/model/ontology/platform.rdf',
-            __DIR__. '/model/ontology/taodelivery.rdf',
```

Comment:
Can't comment on correct line but you probably need to update the extension version

Reply:
We plan to do it once the new feature is done [here](https://github.com/oat-sa/extension-tao-publishing/pull/63/files)

Reply:
I think it's not necessary since the target branch is an integration branch